18_reinforcement_learning.ipynb

```diff
@@ -1925,12 +1925,12 @@
     "        best_weights = model.get_weights()\n",
     "        best_score = step\n",
     "    print(\"\\rEpisode: {}, Steps: {}, eps: {:.3f}\".format(episode, step + 1, epsilon), end=\"\")\n",
-    "    if episode > 50:\n",
+    "    if episode >= 50:\n",
+    "        if episode % 50 == 0:\n",
+    "            target.set_weights(model.get_weights())\n",
```

Comment:
Prior to my `<=` changes, `target.set_weights` was called at `episode == 50` and `training_step(batch_size)` was called at `episode == 51` (i.e. after). I thought we might want to keep that order.